factory/crypto/cryptoComponents.go

```diff
@@ -197,6 +201,13 @@ func (ccf *cryptoComponentsFactory) Create() (*cryptoComponents, error) {
 		return nil, err
 	}
 
+	for _, skBytes := range cp.handledPrivateKeys {
```

Comment:
I think this for loop can be avoided :thinking: 

what if we move L143-L146 on L204 and pass `managedPeersHolder` as parameter on `createCryptoParams` and further on `processAllHandledKeys`, this `managedPeersHolder.AddManagedPeer` can be done on L399

Reply:
It can be avoided but then, the createCryptoParams function will also add the handled key in the managedPeersHolder. This will mean that the createCryptoParams will do more than one thing and I believe the code will lose its readability. Since this operation is only done once, when the node starts, I do not think performance degradation will be an issue.
I rather keep the code as it is for this consideration.